r_arm_cart_twist:
    type: robot_mechanism_controllers/CartesianTwistController
    root_name: torso_lift_link
    tip_name: r_gripper_tool_frame
    ff_trans: 0.0
    fb_trans:
      p: 20.0
      i: 0.5
      d: 0.0
      i_clamp: 1.0
    ff_rot: 0.0
    fb_rot:
      p: 0.5
      i: 0.1
      d: 0.0
      i_clamp: 0.2
l_arm_cart_twist:
    type: robot_mechanism_controllers/CartesianTwistController
    root_name: torso_lift_link
    tip_name: l_gripper_tool_frame
    ff_trans: 0.0
    fb_trans:    
      p: 20.0
      i: 0.5
      d: 0.0
      i_clamp: 1.0
    ff_rot: 0.0
    fb_rot:
      p: 0.5
      i: 0.1
      d: 0.0
      i_clamp: 0.2

